pubsub/topics/publish.go

```diff
@@ -14,7 +14,6 @@
 
 package topics
 
-// [START pubsub_publish]
```

Comment:
Confirming docs are updated to not depend on deleted region tags.

Reply:
Yeah, just confirmed in devrel/samples we only depend on the quickstart ones.